README.md

```diff
@@ -42,6 +42,15 @@ To install the package, use `pip` (you must also have `git` installed):
 
 ```
 pip install https://github.com/google/gcp_scanner/archive/main.zip
+gcp-scanner --help
```

Comment:
Please use rebase to maintain clean commit history.

Reply:
@mshudrak apologies, I am fairly new to git and open source. Where should I rebase back to?

Reply:
I'd suggest to check this blog https://www.atlassian.com/git/tutorials/merging-vs-rebasing. Basically, your commits need to go on top of the one where I introduced this change.

Reply:
This change is still visible.